nixos/doc/manual/release-notes/rl-2311.section.md

```diff
@@ -121,6 +121,8 @@
 
 - `buildGoModule` `go-modules` attrs have been renamed to `goModules`.
 
+- The `fonts.fonts` option has been renamed to `fonts.packages`.
```

Comment:
You should mention `enableDefaultPackages` too, no?

Reply:
derp